ethers-providers/src/transports/quorum.rs

```diff
@@ -556,6 +572,15 @@ where
     }
 }
 
+/// Helper type that can be used to pass through the `params` value.
+/// This is necessary because the wrapper provider is supposed to skip the `params` if it's of
+/// size 0, see `crate::transports::common::Request`
+#[derive(Clone)]
+pub enum QuorumParams {
```

Comment:
don't need to expose probably
```suggestion
enum QuorumParams {
```

Reply:
Sadly we do because the wrapper is also `pub`. Oh, BTW, I have a bone to pick about this because if you `use ethers::prelude::*` it currently means the `JsonRpcClientWrapper` creates a conflict on the `request` function forcing you to call it via `JsonRpcClient::request(...)` or just not `use ethers::preulde::*`,

Reply:
I see,

that's actually a good point.

imo quorum should not be reexported via `prelude::*`